docs/getting-started-on-snowplow-open-source/quick-start-aws/index.md

```diff
@@ -28,11 +28,13 @@ import PocketEdition from "@site/docs/reusable/pocket-edition-pitch/_index.md"
     - [Configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config) against a role that has the AdminstratorAccess policy attached
         - AdminstratorAccess allows all actions on all AWS services and shouldn't be used in production
 - [Terraform 1.0.0](https://www.terraform.io/downloads.html) or higher installed
-    - Follow the instructions to make sure the terraform binary is available on your PATH. You can also use [tfenv](https://github.com/tfutils/tfenv) to help manage Terraform installation
+    - Follow the instructions to make sure the `terraform` binary is available on your `PATH`. You can also use [tfenv](https://github.com/tfutils/tfenv) to help manage your Terraform installation
 - Download [the latest igluctl](/docs/pipeline-components-and-applications/iglu/igluctl-2/index.md) which allows you to publish schemas for your [custom events](/docs/understanding-tracking-design/out-of-the-box-vs-custom-events-and-entities/index.md#custom-events) and [entities](/docs/understanding-tracking-design/predefined-vs-custom-entities/index.md#custom-contexts) to [Iglu (your schema registry)](/docs/pipeline-components-and-applications/iglu/index.md)
```

Comment:
Would remove this from here, I think it’s easier to deploy first and then figure out the usage. Same way as we are not asking to have `psql` installed at this point.